src/ln/channelmanager.rs

```diff
@@ -1685,12 +1686,12 @@ impl ChannelManager {
 		}
 	}
 
-	fn internal_open_channel(&self, their_node_id: &PublicKey, msg: &msgs::OpenChannel) -> Result<(), MsgHandleErrInternal> {
+	fn internal_open_channel(&self, their_node_id: &PublicKey, their_local_features: &Option<LocalFeatures>, msg: &msgs::OpenChannel) -> Result<(), MsgHandleErrInternal> {
```

Comment:
Hmm, why Optional? Protocol should enforce that its always set, no?

Reply:
Yes, after verification it's always set, corrected